src/components/AboutUsIntroduction/index.tsx

```diff
@@ -19,7 +15,7 @@ const AboutUsIntroduction = () => {
   return (
     <section className="about-us-section">
       <div className="about-us-introduction about-us-introduction--background">
-        {renderIntroductionTitle()}
+        <Trans i18nKey={t('introductionSectionTitle')} components={titles} />
         {renderIntroductionDescription()}
```

Comment:
Since you decided decompose the functions in `NewReleaseBanner/index.tsx `, please decompose this function too?